integration-tests/src/docker/artifacts.ts

```diff
@@ -34,19 +35,32 @@ export async function copyRunArtifacts(
     }
   )
 
-  await runShellCommands([`docker logs jahia`], 'artifacts/results/jahia.log', {
-    ignoreReturnCode: true,
-    silent: true
-  })
+    async function getRunningContainerNames(): Promise<string[]> {
+        const output: string = await core.group('List Docker containers', async () => {
+            return await exec.getExecOutput('docker', ['ps', '-a', '--format', '{{.Names}}'], {
+                ignoreReturnCode: true,
+                silent: true
```

Comment:
Are those parameters needed now that we don't target a specific container name? Or are there still cases where this command is expected to fail?

Reply:
The `ignoreReturnCode` has been removed, we want to know when the export fails, `silent` si remaining set to prevent all logs to be part of the output